pkg/apis/core/v1alpha1/register.go

```diff
@@ -48,12 +48,14 @@ func AllResourceObjects() []resource.Object {
 		&Session{},
 		&FileWatch{},
 		&Cmd{},
+		//&KubernetesApply{},
```

Comment:
Did you want to hide these from the API for now?

Reply:
ya, will uncomment once we need controllers :+1: